meson.build

```diff
@@ -120,7 +120,7 @@ geoclue_dep = dependency(
 )
 libportal_dep = dependency(
   'libportal',
-  version: '>= 0.8.1',
+  version: '>= 0.9.0',
```

Comment:
libportal 0.9.0 has not been released yet. If x-d-p needs a newer libportal than 0.8.1, then we need a libportal release first.

Reply:
the thing is:

1. it's only for the test
2. it used to be that it wanted a special branch off the repository and I pushed back. This is currently in master in the official repo.
3. this is meant to trigger the meson subproject. I have 0.8.1 installed and the build fail because of that (it build the tests in the default target).

So we can keep the build broken until a release, or we can just accept it and move forward. Or disable the tests again.

Reply:
Yeah, the projects have a cyclic dependency so depending on an unreleased version is fine, until we want to have a release.